EIPS/eip-dilithium.md

```diff
@@ -0,0 +1,261 @@
+---
+eip: 9999
+title: Precompile for ML-DSA signature verification
+description: Proposal to add precompiled contracts that perform signature verifications using the NIST-standard FIPS-204 ML-DSA and an EVM-friendly variant 
+author: Renaud Dubois (@rdubois-crypto), Simon Masson (@simonmasson)
+discussions-to: https://ethereum-magicians.org/t/add-eip-ml-dsa-verification/25857
```

Comment:
```suggestion
discussions-to: https://ethereum-magicians.org/t/eip-8051-ml-dsa-verification/25857
```
Updated with assigned number

Reply:
Please update the discussions topic to use the template
https://ethereum-magicians.org/c/eips/5